test/end-to-end/util/First-Link-Handle.ts

```diff
@@ -0,0 +1,15 @@
+import { createLinkButton } from './helpers'
+
+/* If there are no links in the account, link button 2 will be present.
+ * If there is at least 1 link in the account, link button 1 will be present.
+ */
+
+const firstLinkHandle = async (t) => {
```

Comment:
nitpick: to use [jsdoc format ](https://javascript.info/comments)for function documentation.

no need to have return types/params, but adopt the correct number of asterisk `*`s, removing the newline on line 6, etc etc